jmeter/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "JMeter",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "jmeter",
+  "metric_prefix": "jmeter.",
+  "metric_to_check": "jmeter.responses_count",
+  "creates_events": false,
+  "short_description": "A Datadog plugin for Apache JMeter",
+  "guid": "73e25799-9bc1-413b-a5f3-989a7c5bd554",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "JMeter",
+  "categories": [
+    "testing"
```

Comment:
This is a new category, but I couldn't find any existing one that would be valid for JMeter. Any thoughts?